django_project/blog/admin.py

```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
+from .models import Post
 
-# Register your models here.
+admin.site.register(Post)
```

Comment:
it's good practice to have a new line at the end of a file, not a a major concern but good practice

Reply:
Here's a link that explains it better than I could https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
That makes sense, Ive added it.